### ISSUE TITLE
Allow modification of SSM-SessionManagerRunShell document

### ISSUE THREAD

**Issue by danieljamesscott** [aws-cloudformation/aws-cloudformation-coverage-roadmap#460]
## Systems manager SSM-SessionManagerRunShell document

This document is 'owned by me' and contains the session manager settings (S3 bucket for audit logs etc)

It's not possible to manage this using cloudformation because the document already exists and cannot be imported into a stack.

**Comment by jk2l** [comment]
Atm i just write a custom resource to manage update. but TBH i don't think this configuration should even be part of SSM document. it should be a AWS account configuration setting like IAM Password Policy

**Comment by benbridts** [comment]
@danieljamesscott Did you try deleting it with the CLI?

**Comment by danieljamesscott** [comment]
No, but that's a good idea. I don't really need this any more, but would be good to know if it works.

**Comment by tdmalone** [comment]
Deleting it with the CLI solves the first problem:
```
aws ssm delete-document --name SSM-SessionManagerRunShell
```

But after you've created it with CloudFormation, you can't make any updates due to:
```
CloudFormation cannot update a stack when a custom-named resource requires replacing. Rename SSM-SessionManagerRunShell and update the stack again.
```
(well, I guess you could rename it to a placeholder name and then rename it back, but that's a pain...)

**Comment by Obirah** [comment]
> --name SSM-SessionManagerRunShell

CloudFormation now has `updateMethod=NewVersion` (see [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-document.html#cfn-ssm-document-updatemethod)) which could solve that problem at least.
